backend/models/customer.js

```diff
@@ -3,15 +3,42 @@ const ClientBase = require("./client");
 
 const Schema = mongoose.Schema;
 
+var address = {
+  addressName: { type: String },
+  name: { type: String },
+  addressLine: { type: String },
+  city: { type: String },
+  state: { type: String },
+  zipCode: { type: String },
+  phone: { type: String },
+};
+
+var shoppingList = {
+  title: { type: String },
+  wishedProducts: [],
```

Comment:
Wished products should be a list of ObjectID, please see below comments about them.

Reply:
Added as:
`wishedProducts: [{ type: Schema.Types.ObjectId }]`